examples/examples.yaml

```diff
@@ -21,3 +21,35 @@ examples:
   - id: "tensorflow-akka"
     path: "tensorflow-akka"
     description: This app shows how to serve a Tensorflow model using Akka
+  - id: "sensor-data-scala-snippets"
```

Comment:
@skyluc we might have to re-assess the usage of this file. It had a long-forgotten purpose at some point but it would be interesting to see whether we still need a manually updated list of example projects.

Reply:
It is used in `scripts/build-all.sh`, which is used in the PR check to build and test all example projects.
It is really practical, I only had to add the new projects to the list, and they became part of the CI.

Reply:
Wouldn't it work just as well to just list the number of sub-dirs or `find` all sbt projects in the structure?